tasks/ci.py

```diff
@@ -0,0 +1,63 @@
+import os
+import re
+
+from github import Auth, Github
+from invoke.tasks import task
+
+COMMIT_TITLE_REGEX = re.compile(r"\[test-infra-definitions\]\[automated\] Bump test-infra-definitions to ([a-z0-9]*)")
+
+
+@task
+def create_pr_and_close_stale_ones(ctx, branch: str, new_commit_sha: str, old_commit_sha: str):
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
def create_bump_pr_and_close_stale_ones_on_datadog_agent(ctx, branch: str, new_commit_sha: str, old_commit_sha: str):
```